tests/test_panels/_utils.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+
+# stdlib
+import re
+import sys
+import unittest
+
+# pyramid testing requirements
+from pyramid import testing
+from pyramid.response import Response
+
+
+# ------------------------------------------------------------------------------
+
+
+# used to ensure the toolbar link is injected into requests
+# copy it onto the testcase so 
+re_toolbar_link = re.compile(r'(?:href="http://localhost)(/_debug_toolbar/[\d]+)"')
+
+
+# for tests
+PY2 = True if sys.version_info.major == 2 else False
+PY3 = not PY2
```

Comment:
this is already in `pyramid_debugtoolbar.compat`

Reply:
fixing.